src/pages/Developers.jsx

```diff
@@ -0,0 +1,113 @@
+
+import LoggedInNavbar from "../components/LoggedInNavbar";
+import rect from "/src/assets/Images/Rect.png"
+import star from "/src/assets/Icons/star-light.png";
+import star2 from "/src/assets/Icons/star-grey.png";
+import timer from "/src/assets/Icons/timer.png";
+import symbols from "/src/assets/Icons/symbols.png";
+import locator from "/src/assets/Icons/locator.png";
+
+import { devTeam } from "../utils/data";
+
+
+const Developers = () => {
+
+
+    return (
+        <section>
+            <LoggedInNavbar />
+
+            <div className=" w-[70%] container mx-auto">
+                <button className="w-[178px] h-[48px] text-[#FFFFFF] text-[16px] font-bold  bg-[#2776EA] mt-[90px] rounded-t-md">Find a Candidate</button>
+                <div className="bg-[#F5F5F5] p-4  flex flex-col md:flex-row  flex-wrap  items-center   gap-5 md:gap-11 ">
+
+                    <div className="relative">
+                        <div className="absolute top-3">
+                            <img src={timer} />
+                        </div>
+                        <select className="px-6 py-4 w-[210px]">
+                            <option className="m-auto">
+                                Product Developer</option>
+
+                        </select>
+                    </div>
+
+                    <div className="relative">
+                        <div className="absolute top-3 left-2">
+                            <img src={symbols} />
+                        </div>
+                        <select className="px-11 py-4  w-[210px]">
+                            <option className="p-9">
+                                All</option>
+
+                        </select>
+                    </div>
+
+                    <div className="relative">
+                        <div className="absolute top-3 left-2" >
+                            <img src={locator} />
+                        </div>
+                        <select className="px-7  py-4  w-[210px]">
+                            <option className="p-9">
+                                Nigeria</option>
+
+                        </select>
+                    </div>
+
+
+                    <button className="bg-[#2776EA] w-[210px] h-[46px]   md:ml-[60px]  text-[#FFFFFF]">Search</button>
+
+                </div>
+
+            </div>
+            <div className="flex w-[80%] container mx-auto mt-7 gap-4">
+                <img className="w-[10px] h-[48px] mt-3" src={rect} />
+                <h2 className=" text-[24px] md:text-[48px] mt-4 md:mt-0 font-bold">Software  Developers</h2>
+            </div>
+
+            <div className="flex flex-wrap  justify-between gap-y-11 lg:gap-[170px] lg:w-[70%] w-[80%] container mx-auto mt-9">
+                {devTeam.map((developers) => {
+
+                    const { id, Image, NAME, experience, LOCATION } = developers
```

Comment:
It's important to be consitent. Small letters are preferable except otherwise stated

Reply:
changes have been made to code